online-shop/src/App.css

```diff
@@ -0,0 +1,79 @@
+.App {
+  text-align: center;
+}
+
+.App-basic {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  justify-content: top;
+  flex-direction: column;
+  font-size: calc(10px + 2vmin);
+}
+.header {
+  background-color: #343a44;;
+  padding-bottom: 40px;
+}
+
+.SameLine {
+  display: flex;
+  justify-content: left;
+  position: relative;
+}
+
+.Title2 {
```

Comment:
Please name the CSS classes more intuitively. Something like ProductDetailsTitle, ProductListTitle, or so